### ISSUE TITLE
about the pre-processing

### ISSUE THREAD

**Issue by chn52979** [kxhit/SG_PR#14]
Hi, I have a question about the pre-processing. In the paper, you process the raw point cloud by segmentation and ignore classes like person, because they are either irrelevant or few in number. So why not ignore the vehicle class, which I think is movable and would interfere with place recognition.

**Comment by kxhit** [comment]
Hi, it's a good point. 
We actually didn't do the ablation study to evaluate which semantic class contributes more or less. I think I can explain from the following aspects.

- We are inspired by SegMap/SegMatch which uses 'cars' for object-level place recognition.
- In fact, KITTI is not a long-term dataset which means that most cars don't move in different revisitings.
- We designed an attention module to learn the weights of each node with results shown in Fig.6. We hope it can make the network learn the different semantics and different distance distribution may have different effects on the final scene representation.
- In our recent evaluation, we found that different scenarios/sequences are affected by different semantics to different degrees.

I think you are encouraged to have a try! Hope my explanation is helpful.
